lambdas/src/MembershipSub-Promotions-Scheduled-Dynamo-Backup.js

```diff
@@ -1,64 +0,0 @@
-import { DynamoDB } from "@aws-sdk/client-dynamodb";
-
-const ddb = new DynamoDB();
-
-function createTodaysBackup(TableName) {
-    return ddb.createBackup({
-        'BackupName': `${TableName}-Scheduled-Backup`,
-        TableName
-    })
-        .then(possiblyJustCreatedBackup => {
-            console.log(`Created backup of table ${TableName}: ${possiblyJustCreatedBackup.BackupDetails.BackupArn}`);
-            return possiblyJustCreatedBackup.BackupDetails.BackupArn;
-        });
-}
-
-function deleteBackupsOlderThanXDays(TableName, retentionDays) {
-    return (latestBackupARN) => {
-        const unixtime = Math.floor(Date.now() / 1000);
-        const retentionSeconds = (retentionDays * 60 * 60 * 24);
-        const timeRangeCutoffTimestamp = unixtime - retentionSeconds;
-        const TimeRangeUpperBound = new Date(timeRangeCutoffTimestamp * 1000);
-
-        return ddb.listBackups({
-            TableName,
-            TimeRangeUpperBound
-        })
-            .then(oldBackups => {
-
-                const backupsToDelete = oldBackups.BackupSummaries.map(backupSummary => backupSummary.BackupArn).filter(backupArn => latestBackupARN !== backupArn)
-
-                console.log(`${backupsToDelete.length} old backups for table: ${TableName} will be deleted...`);
-
-                const deletionPromises = backupsToDelete.map(BackupArn => {
-                    console.log(`Attempting to delete backup: ${BackupArn}`);
-                    return ddb.deleteBackup({ BackupArn });
-                });
-
-                return Promise.all(deletionPromises).then(results => {
-                    results.forEach(result => {
-                        console.log(`Successfully deleted ${result.BackupDescription.BackupDetails.BackupArn}`)
-                    });
-                    console.log(`Successfully deleted ${results.length} backups for table: ${TableName}`);
-                    return `Successfully${latestBackupARN ? ' created new backup and' : ''} deleted ${results.length} backups for table: ${TableName}`;
-                })
-            });
-    };
-}
-
-export const handler = (event, context, callback) => {
-
-    const TOUCHPOINT_BACKEND = /PROD$/.test(context.functionName) ? 'PROD' : 'CODE';
-    const RETENTION_DAYS = 14; // a number < 1 will delete everything except the backup `createTodaysBackup` just created
```

Comment:
The new solution is using the same retention period: https://github.com/guardian/aws-backup/blob/12ccfc4c4cd20c5eebfa2803beaee8521856b86f/cdk/lib/aws-backup.ts#L34